edx_solutions_api_integration/courses/tests.py

```diff
@@ -2018,6 +2018,11 @@ def test_coursemodulecompletions_get_invalid_course(self):
         response = self.do_get(completion_uri)
         self.assertEqual(response.status_code, 404)
 
+    def test_courses_metrics_social_check_service_availability(self):
```

Comment:
Can we add couple of test cases to test scenario when we have `CommentClientRequestError`, `ConnectionError`?